docs/node-operator-questionnaire.md

```diff
@@ -0,0 +1,53 @@
+# Node Operators Questionnaire
+
+Thank you for showing interest in becoming a nilDB Node Operator. This questionnaire helps us understand your operational and infrastructure compatibility. Please provide detailed responses and return to your Nillion point of contact.
+
+## Organization Details
+
+Please provide your organization name (if applicable), primary contact name, role/title, contact email, phone number, and the timezone in which your team primarily operates.
+
+## Infrastructure & Hosting
+
+### Network Configuration & Security
+
+A nilDB node requires a DNS record from a public zone and runs a RESTful API service that should be publicly accessible. The service expects TLS termination upstream on port 443, with internal traffic on port 8080 and metrics exposed on port 9091. Please describe your capabilities for managing DNS records, handling TLS termination (including certificate management and renewal), and implementing network security controls. Include any custom resolver settings or cache policies that might affect DNS record freshness.
+
+### Compute Resources
+
+A nilDB node is packaged as a Docker image supporting Linux/amd64 and aarch64 platforms. Please describe your container orchestration platform, scaling capabilities, and resource management approach. Include details about your infrastructure provider(s) and how you achieve geographic distribution of resources, as the nilDB network requires both host and geo-diversity.
+
+### Data Storage
```

Comment:
Probably a bit difficult to estimate: But if operators' MongoDB's arent' auto-scaled, they may want a projection as to how large NilDB's datasets could be (NilVM operators wanted similar estimates). E.g. should they expect infinite growth and to be prepared to shard?

Reply:
Good pt. I'll update the paragraph to mention scaling storage